### DISCUSSION TITLE
[Q] bump-cask-pr - How to process a cask when it has a Major Upgrade

### DISCUSSION THREAD

**Discussion by andrewcrook** [Homebrew/discussions#3522] [Casks]
`processing` is now on `version 4.0.1`

A live check shows ...

```
>> brew livecheck  processing                                                                                     
processing: 3.5.4,0270 ==> 3.5.4,0270
```

So` processing` needs to be updated to 4 and I suppose a `brew-versions` or `brew-cask` should now have a cask `processing3` ?
I just wondered what the process to follow  is to achieve this. I presume there is a certain way the brew team like to do this?

**Comment by SMillerDev** [comment]
Just modify the cask however you like and send a pull request. We have `brew bump` as a convenience method but obviously that won't work if brew does not detect the update.

**Reply by andrewcrook** [reply]
many thanks. I have submitted the required changes to the cask via pull request [#129797](https://github.com/Homebrew/homebrew-cask/pull/129797)
